nucypher/characters/base.py

```diff
@@ -485,7 +496,30 @@ class - whatever type of object that may be.
         power_up = self._crypto_power.power_ups(power_up_class)
         return power_up.public_key()
 
-    def _set_checksum_address(self):
+    def _set_checksum_address(self, checksum_address=None):
+
+        if checksum_address is not None:
```

Comment:
Can it be `NO_BLOCKCHAIN_CONNECTION` too?

Reply:
Not anymore, I don't think.  It can end up that way.